### DISCUSSION TITLE
Gradient enhanced PINN and multivariate PDEs

### DISCUSSION THREAD

**Discussion by MaAl13** [lululxvi/deepxde#1461] [Q&A]
In the following i have two independent questions. One would be that with great interest i have been reading the paper "Gradient enhanced physics informed neural networks for forward and inverse PDE problems" but don't know how to use Deepxde to implement it. I thought about adding the derived equations to the normal ones, but then i don't know how to add weights.

Another question is how do i use PI-DeepONets to solve PDEs that have more than one equations, so a coupled system essentially. Do i need the DeepM&Mnets, if so how do i implement them in deepxde?

Thanks for any answers that i get in advance

**Comment by lululxvi** [comment]
See https://github.com/lu-group/gpinn, https://github.com/lululxvi/deepxde/pull/1410
